Adding sudo note and minor change to validation output in run-auto-remediation.md

justinjb opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->
- adding a note about editing sudoers to allow sensu user to run commands without a password to avoid errors
- changed sample validation output (specifically the command subsection) to be consistent with what was specified in the creation of the hook

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
Since I encountered the sudo askpass error when setting up auto-remediation, I figured a note on avoiding it was warranted. 

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
I forked the repo, added in my changes, ran **yarn run server** and made sure it all looked good.

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New doc/guide
- [ ] Fixing errata
- [x] Update (Add missing or refresh existing content)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] All tests have passed.

cwjohnston commented:
I noticed an unintended change to the example IP address used in the puppetdb enterprise integration docs but I don't think of consequence.

cwjohnston commented:
Silencing reference doc doesn't describe new `begin` parameter, although it's been added to the API reference.

cwjohnston commented:
Results API reference doc does not illustrate history data as added by 1.2

portertech commented on docs/1.2/enterprise/integrations/puppet.md:
```diff
@@ -0,0 +1,204 @@
+~~~ json
+{
+  "puppet": {
+    "endpoint": "https://10.0.1.22:8081/pdb/query/v4/nodes/",
```
Yeah, I did see this. I did ignore it.

portertech commented:
@cwjohnston thank you for the review! I think I've addressed the issues you raised. We can always tweak things going forward :+1:

justinjb committed:
Update run-auto-remediation.md 

- adding a note about editing sudoers to allow sensu user to run commands without a password to avoid errors
- changed sample validation output (specifically the command subsection) to be consistent with what was specified in the creation of the command

justinjb committed:
Merge branch 'master' into patch-1

apaskulin reviewed (approved):
:tada: Thanks for adding this!
